library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterTest.java

```diff
@@ -400,6 +400,11 @@ public void testRecoveryOfPartialFile() throws InterruptedException
         testMeta.blockMetadataSink.collectedTuples.get(1).getFilePath().endsWith(file2));
   }
 
+  /**
+   * Ignoring this test because {@link FileSplitter} is deprecated. In order to fix some of the race conditions in the
+   * operator, the signature of the operator changed which is why it was deprecated.
+   */
+  @Ignore
```

Comment:
FileSplitterInput which is a replacement of this class, has the same test which I fixed. We can just remove this.